v1.2 made edits for 1.2 release (#130)
mulesoft/docs-service-mesh: modules/ROOT/pages/provision-adapter-configure-service-mesh-CLI.adoc

. To verify that the the applications are redeployed, run the following command:
+
`kubectl get pods -n <namespace>`.

> .. If you are running Anypoint Service Mesh on *Openshift* you need to run the following command on the namespace you are running the applications:

Reviewer: ```suggestion
.. If you are running Anypoint Service Mesh on OpenShift,* run the following command in your command namespace:
```

Author: Done! Thanks!
